ovs/v1/ovs.yaml

```diff
@@ -223,13 +160,126 @@ paths:
           content:
             application/json:
               schema:
-                $ref: '#/components/schemas/transportCall'
+                $ref: 'https://api.swaggerhub.com/domains/dcsaorg/DCSA_OVS_DOMAIN/1.0.0#/components/schemas/transportCall'
```

Comment:
Since you are using the same object here as in the POST body - I would not expect a transportCallID in this object. If there is a transportCallID in this object - is it on purpose that you reuse the object in POST?